### PR TITLE
Dell/Dell10g get vlan with interfaces list

### PR DESCRIPTION
**Pull request by emmurd**
This is a new PR with all the corrections submitted and some enhancements.

### CODE REVIEW

File: netman/adapters/switches/dell.py [hunk 1]
```diff
@@ -107,9 +106,10 @@ def get_vlan(self, number):
         elif regex.match("^ERROR", result[0]):
             raise UnknownVlan
         else:
-            vlan = parse_vlan_list(result)
             while len(result) > 0 and "--More--" in result[-1]:
-                result = self.shell.send_key("m", wait_for=("--More-- or (q)uit", "#"), include_last_line=True)
+                result += self.shell.send_key("m", wait_for=("--More-- or (q)uit", "#"), include_last_line=True)
+
+            vlan = parse_vlan_list(result)
```
Comment (lindycoder) [inline netman/adapters/switches/dell.py]: This var should be "vlan**s**" with an s unless you move the [0] on this line

Comment (idjaw) [inline netman/adapters/switches/dell.py]: Should it be "show vlan" or "show vlans"?

**Comment by fbouliane** [comment]
FYI this is a reopen from PR #81 , :+1: for me !

**Comment by lindycoder** [comment]
:+1:

**Pull request merged**